.github/workflows/pr.yml

```diff
@@ -0,0 +1,58 @@
+name: "PR Tests"
+permissions: read-all
+
+# Trigger for PR an merge to develop branch
+on:
+  push:
+    branches: develop
+  pull_request:
+  workflow_dispatch:
+
+env:
+  CTEST_OUTPUT_ON_FAILURE: 1
+  LAPACK_VERSION: 3.12.0
+  PARALLEL: -j 2
+
+jobs:
+  mkl:
+    runs-on: intel-ubuntu-latest
```

Comment:
We run CPU tests on Red Hat 9 internally. If there are no issues, we could use this as an extended testing.

Reply:
This is public testing for PR's. Non-intel contributors and reviewers will be able to see and correct problems before they are merged. It is limited by compute resources available: cpu only, single core runners, etc. The intel testing is needed to provide extensive coverage. but requires intel people to look at the results.

Reply:
I understand the scope and purpose of this public CI. I was thinking it might be a little confusing that we list only Red Hat 9 in [the supported operating systems](https://github.com/oneapi-src/oneMKL?tab=readme-ov-file#supported-operating-systems) for CPU but run tests on Ubuntu.

Reply:
Thanks for clarifying.